src/components/GroupPage/components/CreatePostModal.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { withRouter } from 'react-router'
+import { savePost } from '../../../actions/events';
+import ReactModal from 'react-modal';
+
+ReactModal.setAppElement('#root');
+
+class CreatePostModal extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            showModal: false,
+            postContext: ''
+        };
+
+        this.handleOpenModal = this.handleOpenModal.bind(this);
```

Comment:
if you will use arrow functions you don't need to bind actions!

Reply:
I know about this, but used official example, and didn't change it